src/powershell/install-chocolatey.ps1

```diff
@@ -0,0 +1,21 @@
+# Download and execute Chocolatey install script
+Set-ExecutionPolicy Bypass -Scope Process -Force
+[System.Net.ServicePointManager]::SecurityProtocol = [System.Net.ServicePointManager]::SecurityProtocol -bor 3072
+iex ((New-Object System.Net.WebClient).DownloadString('https://community.chocolatey.org/install.ps1'))
+
+# Configure chocolatey to use FireEye's package repository
+iex "choco sources add -n=fireeye -s https://www.myget.org/F/fireeye/api/v2 --priority 1"
+iex "choco feature enable -n allowGlobalConfirmation"
+iex "choco feature enable -n allowEmptyChecksums"
```

Comment:
We always prefer long options if they are available.

```suggestion
iex "choco sources add --name fireeye --source https://www.myget.org/F/fireeye/api/v2 --priority 1"
iex "choco feature enable --name allowGlobalConfirmation"
iex "choco feature enable --name allowEmptyChecksums"
```

Reply:
Resolved here https://github.com/cisagov/windows-commando-vm-packer/pull/21/commits/927713c1dc2605a9440f24839ce53cc90360e382